src/security/cert_generators/file/certificate_db.cc

```diff
@@ -107,12 +105,13 @@ Ssl::Locker::Locker(Lock &aLock, const char *aFileName, int aLineNo):
 
 Ssl::Locker::~Locker()
 {
+    (void)lineNo;
```

Comment:
Please remove this line if it is not needed. If it is needed, we may need a comment to clarify what is going on here.
```suggestion
```

Reply:
It is needed. That variavble is never accessed anywhere, and the compiler complains about it. But there is value in having that information in case of debugging needs, although the only way to access it is via a debugger. And unlike the implicit-fallthrough case, I can see the value in checking whether a member variable is used anywhere, as it is a potential bug.
I'll add a comment